modules/conductor/src/main/java/org/opencastproject/event/handler/OaiPmhUpdatedEventHandler.java

```diff
@@ -367,110 +123,59 @@ public void handleEvent(AssetManagerItem.TakeSnapshot snapshotItem) {
     }
 
     //An episode or its ACL has been updated. Construct the MediaPackage and publish it to OAI-PMH.
-    logger.debug("Handling {}", snapshotItem);
+    logger.info("Handling update event for media package {}",
+            snapshotItem.getMediapackage().getIdentifier().compact());
 
     // We must be an administrative user to make a query to the OaiPmhPublicationService
     final User prevUser = securityService.getUser();
     final Organization prevOrg = securityService.getOrganization();
 
     try {
```

Comment:
Please check whether the media package has been published to OAI-PMH at all before starting to identify elements that would need to be republished if it was

Reply:
Hmm, disagree. The handling of message broker events has nothing to do with the OAI-PMH publications. So if the media package does not meet the criteria for republication, the (message broker) event will be handled anyway.